src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -224,17 +225,28 @@ class JesBackend extends Backend with LazyLogging {
   def generateJesOutputs(backendCall: BackendCall): Seq[JesOutput] = {
     val wdlFileOutputs = backendCall.call.task.outputs flatMap { taskOutput =>
       taskOutput.expression.evaluateFiles(backendCall.lookupFunction, new NoFunctions, taskOutput.wdlType) match {
-        case Success(wdlFiles) => wdlFiles.map(_.valueString)
+        case Success(wdlFiles) =>
+          val x = wdlFiles map mapWdlValueGcsPath
+          val y = x map (_.valueString)
+          y
```

Comment:
not very helpful variable names, and the second one seems unnecessary (just return the expression)